docs/UserGuide.adoc

```diff
@@ -155,15 +155,52 @@ Format: `editcard INDEX_OF_CARD [q/QUESTION] [a/ANSWER]`
 
 Examples:
 
-* `list` +
-Displays all decks. +
-* `edit 1 n/My Deck` +
-Edits the name of the first deck in the list to be `My Deck`.
+* `edit 1 d/1 a/A` +
+Edits the answer of the first card in the first deck to be `A`.
 
 === Sort current list by name : `sortname`
 If user is currently in a deck, sort all card in alphabetical order. +
 Format: `sortname`
 
+=== Locate a deck by name : `find`
+
+Finds decks with names that contain any of the given keywords.
+Format: `find KEYWORD [MORE_KEYWORDS]`
```

Comment:
There used to be a notation for a variable amount of arguments but I took it out because there wasn't any use case before. 

"Items with …​ after them can be used multiple times including zero times e.g. [t/TAG]…​ can be used as   (i.e. 0 times), t/friend, t/friend t/family etc."